server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy

```diff
@@ -669,86 +671,51 @@ class ESSearchService{
       }
 
       if( !errors && exactQuery.hasClauses() ) {
-        Client esclient = ESWrapperService.getClient()
-        SearchRequestBuilder es_request =  esclient.prepareSearch("exact")
 
-        es_request.setIndices(grailsApplication.config.gokb.es.indices.values() as String[])
-        es_request.setTypes(grailsApplication.config.globalSearch.types)
-        es_request.setQuery(exactQuery)
+        SearchRequest searchRequest = new SearchRequest(grailsApplication.config.searchApi.indices.values() as String[])
```

Comment:
The correct config value here should be `grailsApplication.config.gokb.es.indices.values()` instead.